### DISCUSSION TITLE
Github Classroom Ghost Account

### DISCUSSION THREAD

**Discussion by TrentonWallis** [community/community#50644] [GitHub Education]
### Select Topic Area

Question

### Body

I have a student who is using their Github account just find on Github, but recently when logging into github classroom they appear as the deleted ghost account and cannot accept assignments correctly. 

We have tried:

1. logging out of both Classroom and Github, then logging back into Github, then accepting the assignment in classroom 
2. logging out of Classroom then back into Classroom 

The student claims they did not delete their Github account, which seems true since they do not appear as ghost when logged into Github, only when logged into Classroom. 

Does anyone have any insight on this? I am happy to provide more info if needed.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @TrentonWallis ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Victor-Morel** [comment]
Hi, exactly the same issue on my side when testing the setup for my course with a dummy account.

**Comment by weitsang** [comment]
**Bumping this up** A few students in my class are facing exactly the same issue.

**Comment by elonerajeev** [comment]
Hey @savrum  I'm currently working on #LearnTube Software , an awesome software for students (#####), and I'm eager to contribute it to the GitHub Student Developer Pack for free use. Does anyone know if there are specific requirements or guides for this? Thanks a bunch!

**Comment by dcowell** [comment]
I have just had the same issue here. Students is listed as ghost after accepting the assignment. They have even tried creating a new GitHub account for which the same thing happened again.

**Comment by alsheroziy** [comment]
Hello everyone, I just joined the github community, please tell me the rules

**Comment by MaelImhof** [comment]
I ran into the same issue, I did not solve it properly speaking but here is what I learned from it :

I'm a student, I needed to register in a GitHub Classroom, but I kept getting the following result :
![image](https://github.com/community/community/assets/46901347/c83a3619-1a02-48b8-98ac-249ccb42e6b2)

I contacted my teacher and we went through the registered students identifiers and the roster. My identifier was linked to [Ghost](https://github.com/ghost), a user account that is supposed to replace the ones that were deleted. In the same way, instead of my GitHub name (say `GitHubUser`), all assignments were given to and created with the name of Ghost.

Turns out, my GitHub account (`GitHubUser`) did not exist, or so did GitHub Classroom think. I was able to work normally with it from github.com, creating repositories, logging out, logging back in, all normal. But my account did not exist in the eyes of GitHub Classrooms.

I changed accounts and it worked, but I don't know what changed and what made that the second account existed. Maybe this might help someone though.

**Comment by r0b3rtb03** [comment]
I'm having this same issue, and my homework assignment is due and I tried to get my professor to help me out but he was super **unhelpful** and basically blamed me for my problem, even though I've done every assignment so far BUT the one the last one that's due!

**Comment by dkxb** [comment]
I had the same issue, I fixed it by deleting the assignment that is linked to the account named "ghost", logging out, loggin in again, and reaccepting the homework.

**Comment by imgfb** [comment]
No answer, where is github support team? NO ONE HELPING
Classroom is useless, more problems than solutions
A month with a ticket without answer. I've had enough.

Solution for this is create a new account everytime this happens
Beacuse github flags everything as spam so u must spam more to get access

DUMB

**Comment by hifigraz** [comment]
Bump here, same problem with student

**Comment by chowmn** [comment]
Still an issue with my students

**Comment by imam-hasan-jami** [comment]
facing the same problem but the github support team hasn't answered the question yet! disgusting.
![CleanShot 2025-04-08 at 19 46 12@2x](https://github.com/user-attachments/assets/5d827cf6-f56c-44ab-af5a-c0f9cda16f6a)

**Comment by VMorHSE** [comment]
When this issue occurs, GitHub Classroom typically creates a repository named <assignment-name>-ghost. To resolve it:

1. First, manually delete the incorrect repository from GitHub
2. Then ask the student to accept the assignment again using the original invitation link
3. If the student sees a "Reaccept" option, they should use it

This approach has worked successfully in my experience.
